_events/sr2020/london-kickstart.md

```diff
@@ -0,0 +1,57 @@
+---
+title: London Kickstart
+date: 2019-10-26 10:00:00
+layout: event
+type: kickstart
+location: Thread, Whitechapel
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+**There is limited space at the London Kickstart, please only attend the London
+Kickstart if we have informed you that you have a place there.**
+
+If you are unable to attend Kickstart, please [let us know][teams-contact] so we
+can arrange another way to get you your kit.
+
+**Please bring your laptops!**
+
+## Directions
+
+The London Kickstart is being held in the [offices of Thread][venue-map] at:
+
+1 Alie Street,
+Whitechapel,
+London,
+E1 8DE
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
+
+<!-- TODO: find out whether we can provide parking
+If you intend to travel by road, please [let us know][teams-contact] so we can
+arrange for parking.
+ -->
```

Comment:
In theory I think we can offer parking, though this wasn't an issue last year. I don't think that anyone would really want to drive anyway. I suggest dropping these lines and then sorting it out if someone asks about parking.